### PR TITLE
TA-459 Education pricing feature

### PR DESCRIPTION
**Pull request by solomonahmed**
## What does this pull request do?

Create new feature to test pricing and escape fee flag of Education claims

## Why make these changes?

Mandatory Gateway work TA-332

## Checklist

- [ ] Provided link to a JIRA ticket: https://dsdmoj.atlassian.net/browse/TA-459

### CODE REVIEW

File: features/education_outcome_pricing.feature [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    Scenario Outline: Education outcome pricing
+        When user is looking at outcome "<UFN>"
+        Then user should see the outcome with one of these stage reached codes:
+            | EA |
+            | EB |
+            | EC |
+            | ED |
+        And user should see the outcome priced at "£ <Price>"
+        And the Escape Fee flag is "<Escape Fee>"
```
Comment (sldblog) [inline features/education_outcome_pricing.feature]: @Katherineodoherty What do you think?

File: features/education_outcome_pricing.feature [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    • If Profit Costs ex. VAT + Counsel Costs ex. VAT == £816 AND VAT_INDICATOR = 'Y'
+    then Price = Fixed Fee £272 + VAT + (Disbursements + VAT)
+    • If Profit Costs ex. VAT + Counsel Costs ex. VAT == £816 AND VAT_INDICATOR = 'N'
+    then Price = Fixed Fee £272 + (Disbursements + VAT)
+    • If Profit Costs ex. VAT + Counsel Costs ex. VAT > £816 AND VAT_INDICATOR = 'Y'
+    then Price = Fixed Fee £272 + VAT + (Disbursements + VAT) & flag as Escape Fee case
+    • If Profit Costs ex. VAT + Counsel Costs ex. VAT > £816 AND VAT_INDICATOR = 'N'
+    then Price = Fixed Fee £272 + (Disbursements + VAT) & flag as Escape Fee case
+
+    Background: User bulk loads a file with Education outcomes
```
Comment (sldblog) [inline features/education_outcome_pricing.feature]: What do you think about the following?
```suggestion
    Background: The provider firm bulk submits case claims with Education outcomes
```

File: features/education_outcome_pricing.feature (hunk 1 above)
Comment (Katherineodoherty) [reply features/education_outcome_pricing.feature]: Who is the user here? The provider or the escaped fee's team?

Comment (your) [reply features/education_outcome_pricing.feature]: @Katherineodoherty With user we mean whoever is logged in with the roles to access to submit new outcomes, so it should the _provider_ who's seeing the details right after the submission.

Comment (your) [reply features/education_outcome_pricing.feature]: Anyone with the right roles can access that information though, for instance someone from the escape fee's team – that's probably why it's kept a bit generic as _user_. Do you reckon it's not clear enough and we should opt for _provider_ instead?

File: features/education_outcome_pricing.feature (hunk 2 above)
Comment (your) [reply features/education_outcome_pricing.feature]: @sldblog I've created https://github.com/ministryofjustice/laa-cwa-feature-tests/issues/57 so we keep track of this. At the moment there are a few open WIP branches so changing the wording now might be a bit problematic. 😢

File: features/step_definitions/pricing_outcome_steps.rb [hunk 3]
```diff
@@ -80,6 +80,11 @@
   end
 end
 
+Then("user should see the outcome with one of these stage reached codes:") do |table|
+  stage_reached_codes = table.raw
+  stage_reached_codes.include?(@current_outcome.stage_reached.text)
```
Comment (your) [inline features/step_definitions/pricing_outcome_steps.rb]: Is this missing an expectation?

**Comment by vimalsuresh** [comment]
A ticket is raised to look at this issue. https://dsdmoj.atlassian.net/browse/TA-2623

**Pull request merged**